### DISCUSSION TITLE
Exclussive file open & Adler32

### DISCUSSION THREAD

**Discussion by jfilaa** [WerWolv/ImHex#994] [Feedback]
Hello, ImHex is very usefull tool. I have three points to improve it. :-)

Offer non exclussive open of files and listen if file have been changed (as for example Notepad++).
Add Adler32 for analyze of ZLib Files.
Option for change font size.

Thank You for development ImHex.

**Comment by Calinou** [comment]
>  Option for change font size.

[This is already feasible if you set a custom font in the Settings.](https://docs.werwolv.net/imhex/common/fonts) The default font cannot have its size adjusted, as it is meant to be drawn at integer scales of its design size (since it's a pixel art font).

**Comment by Calinou** [comment]
> Add Adler32 for analyze of ZLib Files.

This is supported as of [1.32.2](https://github.com/WerWolv/ImHex/releases/tag/v1.32.2):

> - Merged in old ExtraHashes plugin
>   - This adds support for about 20 new hashes such as Adler32, Blake, Tiger and Murmur

**Comment by robertstefan** [comment]
Voted for the
> non exclusive open of files and listen if file have been changed

feature. Would love to see this behaviour.

**Reply by Calinou** [reply]
Please don't bump issues without contributing significant new information. Use the :+1: reaction button on the first post instead.
